modules/basis_universal/config.py

```diff
@@ -1,5 +1,6 @@
 def can_build(env, platform):
     env.module_add_dependencies("basis_universal", ["jpg"])
+    env.module_add_dependencies("basis_universal", ["tinyexr"])
```

Comment:
```suggestion
    env.module_add_dependencies("basis_universal", ["jpg, tinyexr"])
```

Reply:
My bad, my suggestion had a typo, it should have been:
```
["jpg", "tinyexr"]
```
I'll fix it up and squash.